examples/certgen.py

```diff
@@ -45,14 +45,14 @@ def createCertRequest(pkey, digest="sha256", **name):
     req = crypto.X509Req()
     subj = req.get_subject()
 
-    for (key,value) in name.items():
+    for (key,value) in list(name.items()):
```

Comment:
Why do you convert them into a list? Please don’t. :)

Also let’s just write `key, value` instead of `(key,value)`.